### DISCUSSION TITLE
Ressurskontor - Dashboard

### DISCUSSION THREAD

**Discussion by Remi749** [Puzzlepart/prosjektportalen365#1024] [Ideas]
## Portefølje:

- Ressurskontor - Dashboard (SitePage)

_Som ressurseier har jeg et behov for å holde oversikt på mine ressurser og deres allokering/ tilgjengelighet_

- Dashboardet inneholder filtrerte visninger med "Mine ansatte"
  - De ansatte legges inn i "Ressurser" listen og kobles mot deg som eier og enhet
- DetailsList filtrert på ressursene dine
  - Behøver å kunne legge på kapasiteten til den ansatte (den faste varige)
  - "Linjeoppgaver mine ansatte" Jeg må kunne allokere "linjetid" slik som ferie, permisjon, samt andre linjeoppgaver utenfor prosjektarbeid.
- **"Prosjektforespørsler mine ansatte"** skal synliggjøres slik at jeg kan vurdere og godkjenne/ avvise alt etter behov.
  - Liste med enkel godkjenn/avvis direkte på listen
- Tidslinje for ressursene dine
  - Kort: Bør inneholde informasjon om Identifikator + Navn på Linje/Ide/Prosjekt, start, stopp, belastning, status (forespurt/ godkjent)
- Ressurser (List)
  - Felter:
    - Navn
    - Kapasitet (stillingsprosent)
    - Eiertilhørighet (ansvarlig person)
    - Enhet (ansvarlig persons enhet)

## Prosjekt:
- Ressursallokering (SitePage), listen fases bort
  - Lik som Tidslinje siden for prosjekt
  - Tidslinjevisning med detailslist
    - Detailslisten henter elementer fra porteføljeliste filtrert på prosjektet du står i
    - Kolonner er likt, men samme mulighet for dynamisk

---

### Prosjekt:
- Ressursallokering fra Prosjekt fases bort og blir hidden, innhold migreres til portefølje listen. 

### Portefølje:
- Ressursallokering (Alle ressurser)
  - Migrert fra alle eksisterende prosjekter
  - Kolonner:
    - Prosjektilknytning
    - Ressurs (Navn) -> Lookup fra Ressurser listen

Det overstående er ikke satt i stein og det kan forekomme endringer

Estimater finnes i assist sak 4225

**Comment by olemp** [comment]
@Remi749 @tarjeieo Konvertert til diskusjon da denne ikke er avklart.
